packages/gafl-webapp-service/src/processors/date-and-time-display.js

```diff
@@ -11,13 +12,15 @@ export const advancePurchaseDateMoment = permission =>
  * @param permission
  * @returns {string}
  */
-export const displayStartTime = permission => {
+export const displayStartTime = (permission, displayTimeFirst) => {
```

Comment:
So, if you omit displayTimeFirst, it'll default to `undefined` which evaluates to false. I think that it's worth putting a default value here of false, so that your code's a little bit clearer.